protocol-designer/src/assets/localization/en/shared.json

```
@@ -2,26 +2,65 @@
   "add": "add",
   "amount": "Amount:",
   "cancel": "Cancel",
+  "close": "Close",
   "confirm_import": "Are you sure you want to upload this protocol?",
   "confirm_reorder": "Are you sure you want to reorder these steps, it may cause errors?",
   "confirm": "Confirm",
   "create_a_protocol": "Create a protocol",
   "create_new_protocol": "Create new protocol",
   "done": "Done",
+  "edit_existing": "Edit existing protocol",
   "edit": "edit",
   "eight_channel": "8-Channel",
   "exit": "exit",
   "go_back": "Go back",
-  "import_existing": "Import existing protocol",
   "import": "Import",
+  "incorrect_file_header": "Invalid file type",
+  "incorrect_file_type_body": "Protocol Designer only accepts JSON files of protocols that were previously created in Protocol Designer. Upload a valid file to continue.",
+  "invalid_json_file_body": "This file is either missing information it needs to import properly or contains sections that Protocol Designer cannot read. At this time we do not support JSON files created outside of Protocol Designer.",
```

Another one we can tighten up.
```suggestion
  "invalid_json_file_body": "This JSON file is either missing required information or contains sections that Protocol Designer cannot read. At this time we do not support JSON files created outside of Protocol Designer.",
```